src/routes/Edit/VideoSplitter.tsx

```diff
@@ -593,10 +689,57 @@ const VideoSplitter = ({
                   <p>
                     <br />
                     <Icon name="help circle" />
-                    Tip : Type a phrase and press Tab key to commit.
+                    <small>Tip : Type a phrase and press Tab key to commit.</small>
                     <br />
                   </p>
                 </Grid.Column>
+              </Grid.Row>*/}
+
+              {/*Experimental tags (not to be merged at this time)*/}
```

Comment:
Haha, looks like we are on the same page. Since we are not merging this feature we will need to remove if from this branch/PR and move it to it's own feature branch.

Reply:
Yes, I wanted you to see it first before merging. But now it is no more experimental.